health-sdk/sdk/src/doc/source/flow.rst

```diff
@@ -1,58 +1,169 @@
+
+.. code-block:: kotlin
+
+    paymentComponent.listener = object: PaymentComponent.Listener {
+            override fun onMoreInformationClicked() {
+                // Show the MoreInformationFragment.
+            }
 
-Display ReviewFragment
+            override fun onBankPickerClicked() {
+                // Show the BankSelectionBottomSheet.
+            }
+
+            override fun onPayInvoiceClicked(documentId: String) {
+                // Show the ReviewFragment.
+            }
+        }
+
+Load payment providers
 ----------------------
 
-ReviewFragment displays document pages and extractions and it lets the user pay using a payment provider
+Call ``paymentComponent.loadPaymentProviderApps()`` to load the available payment providers from the Gini Health API and
+to check which ones are installed on the user's device.
+
+.. note::
+
+    It should be sufficient to call ``paymentComponent.loadPaymentProviderApps()`` only once when your app starts.
+
+Show the PaymentComponentViews
+------------------------------
+
+The ``PaymentComponentView`` is a custom view widget and the main entry point for users. It allows them to pick a bank
+and initiate the payment process. In addition, it also allows users to view more information about the payment feature.
+
+The ``PaymentComponentView`` is hidden by default and should be added to the layout of each invoice item:
+
+.. code-block:: xml
+
+    <net.gini.android.health.sdk.paymentcomponent.PaymentComponentView
+        android:id="@+id/payment_component"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        ... />
+
+When creating the view holder for the invoice item, pass the ``PaymentComponent`` instance to the view holder:
+
+.. code-block:: kotlin
+
+    val paymentComponentView = view.findViewById(R.id.payment_component)
+    paymentComponentView.paymentComponent = paymentComponent
 
-To instantiate it you need to create a ``FragmentFactory`` and set it to fragment manager:
+When binding the view holder of the invoice item, prepare it for reuse, set the payable state and the document id:
 
 .. code-block:: kotlin
 
-    class ReviewFragmentFactory(private val giniHealth: GiniHealth,
-                                private val configuration: ReviewConfiguration,
-                                private val listener: ReviewFragmentListener
-    ) : FragmentFactory() {
-        override fun instantiate(classLoader: ClassLoader, className: String): Fragment {
-            return ReviewFragment(giniHealth, configuration, listener)
+    viewHolder.paymentComponentView.prepareForReuse()
+    viewHolder.paymentComponentView.isPayable = invoiceItem.isPayable
+    viewHolder.paymentComponentView.documentId = invoiceItem.documentId
+
+.. note::
+
+    The ``PaymentComponentView`` will only be visible if its ``isPayable`` property is ``true``.
+
+Show the MoreInformationFragment
```

Comment:
[Suggestion] Would it be worth mentioning here that the fragment does not handle any navigation-related events? Such as setting a title, hiding options menu or anything of the sort? 

This needs to be done by the developer (as well as setting the privacy policy link - but I think this will be specified in the customization part of the documentation).